src/ui/hooks/recordings.ts

```diff
@@ -421,11 +421,12 @@ export function useIsOwner() {
   return userId === recording.owner.id;
 }
 
-export function useGetPersonalRecordings():
-  | { recordings: null; loading: true }
-  | { recordings: Recording[]; loading: false } {
+export function useGetPersonalRecordings(
+  filter: string
```

Comment:
optional?

Reply:
Does it need to be optional? I figured since it defaults to "", it should be okay to just pass it in as either an empty string or a real filter string, and then just handle the no filter case in the backend.

Reply:
It's optional on the backend so it makes sense to mark it as such here too, I think. It doesn't really matter particularly because we won't encounter errors by being more restrictive on the front end.